ref/src/CMakeLists.txt

```diff
@@ -3,10 +3,16 @@ netCDFModule.f90
 print_affinity.c
 runnable.c
 yppm_core.F90
+interpolate.f90
 yppm_driver.f90
 )
 
+list (APPEND test_src_files
+interpolateTest.f90
+)
+
 add_executable( ${PROJECT_NAME} ${yppm_src_files} )
+add_executable (interpolateTest ${test_src_files}  interpolate.f90)
```

Comment:
Same with this.  Needs to go in `ref/test/CMakeLists.txt`

Reply:
Done.